src/xtgeo/grid3d/_gridprop_import_eclrun.py

```diff
@@ -474,7 +471,7 @@ def section_generator(generator):
             return
 
 
-def import_gridprops_from_restart_file_sections(
+def find_gridprops_from_restart_file_sections(
```

Comment:
Pedantic alert (!)

"Imports list of parameters from an sections generator."

"Finds list of parameters from a sections generator."

Reply:
Fixed